src/checkoutservice/main.go

```diff
@@ -192,6 +168,10 @@ func (cs *checkoutService) Check(ctx context.Context, req *healthpb.HealthCheckR
 	return &healthpb.HealthCheckResponse{Status: healthpb.HealthCheckResponse_SERVING}, nil
 }
 
+func (cs *checkoutService) Watch(req *healthpb.HealthCheckRequest, srv healthpb.Health_WatchServer) error {
+	return nil
+}
```

Comment:
Is this related to removing jaeger?

Reply:
sorry no, i was getting a build error about it so I added it in there and it worked as a bandaid but is probably not the issue. I'm going to rebase and try again without it, will open a separate pr if it's actually necessary

Reply:
I was clearly confused 😅  just removed all of them